vm/vm.go

```diff
@@ -209,10 +217,10 @@ func (v *vm) Execute(txns []core.Transaction, declaredClasses []core.Class, bloc
 	C.free(unsafe.Pointer(chainID))
 
 	if len(context.err) > 0 {
-		return nil, nil, errors.New(context.err)
+		return nil, nil, nil, errors.New(context.err)
 	}
 
-	return context.actualFees, context.traces, nil
+	return context.actualFees, context.traces, context.reverts, nil
```

Comment:
instead of adding more return values to `Execute()`, let's define a new custom error type and return that instead.
If needed, rpc.Handler or whoever is calling the VM can check for that specific error type instead of treating all errors the same.